src/index.ts

```diff
@@ -120,27 +123,43 @@ app.get('/tests/present/:name', (c) => {
     throw new HTTPException(404, { message: 'test not found' });
   }
 
-  const { credential, presentationFrame } = presentTestCases[name];
+  const { credential, presentationFrame, kb } = presentTestCases[name];
 
-  return c.json({ results: { description, credential, presentationFrame } });
+  return c.json({ results: { description, credential, presentationFrame, kb } });
 });
 
 app.post(`/tests/present/:name`, async (c) => {
   const name = c.req.param('name') as keyof typeof presentTestCases;
+
   if (!presentedNames.includes(name)) {
     throw new HTTPException(404, { message: 'test not found' });
   }
 
-  const sdjwt = await getSDJwt();
-
-  const { claims } = presentTestCases[name];
-
   try {
+    // kb : key bind
+    const { claims, kb: kbQuestion } = presentTestCases[name];
     const { answer } = await c.req.json(); //get a token
-    await sdjwt.validate(answer);
+    const sdjwt = await getSDJwt();
+
+    if (!kbQuestion) {
+      //normal test
+      const { payload } = await sdjwt.verify(answer, undefined, true);
```

Comment:
```suggestion
      const { payload } = await sdjwt.verify(answer);
```

if kbQuestion is not exist(or falsy) then we should not check the key binding in verify